spec/lib/guard/puma/runner_spec.rb

```diff
@@ -16,39 +15,61 @@
     end
   end
 
-  %w(halt restart).each do |cmd|
+  %w[halt restart].each do |cmd|
     describe cmd do
-      before do
-        allow(runner).to receive(:build_uri).with(cmd).and_return(uri)
+      context "without pumactl" do
+        let(:options) { { pumactl: false } }
+
+        let(:uri) do
```

Comment:
I'm a proponent of the [Semantic Rule](https://github.com/bbatsov/ruby-style-guide/issues/162) for `{}` vs `do/end` so for the `let`'s I prefer to use `{}`.

Reply:
Heh, I didn't mean to make this comment twice. I think it came from having 2 browser tabs open for reviews of different commits. Oops!

Reply:
There were many places with `let(:foo) do; end`, I replaced all of them.